src/sdk/modules/validators/ownableValidator/toOwnableValidationModule.ts

```diff
@@ -0,0 +1,59 @@
+import { getOwnableValidatorMockSignature } from "@rhinestone/module-sdk"
+import type { Account, Client, Hex, Prettify } from "viem"
+import { toSigner } from "../../../account"
+import { toValidationModule } from "../toValidationModule"
+import type { Module, ModuleImplementation } from "../types"
+
+export type ToOwnableValidatorModuleReturnType = Prettify<
+  Module<OwnableValidatorModuleImplementation>
+>
+
+export type OwnableValidatorModuleImplementation = ModuleImplementation
+
+export const toOwnableValidatorModule = async ({
+  address,
+  client,
+  initData,
+  deInitData,
+  threshold
+}: {
+  address: Hex
+  initData: Hex
+  deInitData: Hex
+  client: Client
+  threshold: number
+}): Promise<ToOwnableValidatorModuleReturnType> => {
+  const signer = await toSigner({ signer: client.account as Account })
+  return toValidationModule({
+    signer,
+    address,
+    initData,
+    deInitData,
+    getDummySignature: () => {
+      return getOwnableValidatorMockSignature({ threshold })
+    },
+    signUserOpHash: async (userOpHash: Hex) => {
+      const signature = await signer.signMessage({
+        message: { raw: userOpHash as Hex }
+      })
+      return signature as Hex
+    },
+    signMessage: async (_message: Uint8Array | string) => {
+      const message =
+        typeof _message === "string" ? _message : { raw: _message }
+      let signature = await signer.signMessage({ message })
+
+      const potentiallyIncorrectV = Number.parseInt(signature.slice(-2), 16)
```

Comment:
Let's stick this function somewhere and just reuse it.

Reply:
Fixed